Sources/Validation/Convenience/StrongPassword.swift

```diff
@@ -0,0 +1,27 @@
+private let regex = "^.*(?=.{3,})(?=.*[a-zA-Z])(?=.*[0-9])(?=.*[\\d\\X])(?=.*[^a-zA-Z\\d\\s:]).*$"
+
+/*The password contains characters from at least three of the following four categories:
+
+ English uppercase characters (A – Z)
+ English lowercase characters (a – z)
+ Base 10 digits (0 – 9)
+ Non-alphanumeric (For example: !, $, #, or %)
+ Unicode characters
+ */
+
+public struct StrongPassword: Validator {
```

Comment:
"strong" is like tanner said, controversial. The length usually matters more than the randomness. And randomness usually results in reusing passwords. So this is more often than not counter-productive with good intentions 😬 

I'd love it if this would be a `Password` validator with configuration options. Allowing definition of a minimum length, some optional "entropy" checks, maximum reoccurring characters check, etc. That's likely too complex for this PR, but that would make validation awesome.